keras_cv/models/object_detection/yolox/__internal__/layers/yolox_head.py

```diff
@@ -0,0 +1,151 @@
+                    kernel_size=1,
+                    strides=1,
+                    activation=activation,
+                )
+            )
+
+            self.classification_convs.append(
+                keras.Sequential(
+                    [
+                        ConvBlock(
+                            filters=int(256 * width_multiplier),
+                            kernel_size=3,
+                            strides=1,
+                            activation=activation,
+                        ),
+                        ConvBlock(
+                            filters=int(256 * width_multiplier),
+                            kernel_size=3,
+                            strides=1,
+                            activation=activation,
+                        ),
+                    ]
+                )
+            )
+
+            self.regression_convs.append(
+                keras.Sequential(
+                    [
+                        ConvBlock(
+                            filters=int(256 * width_multiplier),
+                            kernel_size=3,
+                            strides=1,
+                            activation=activation,
+                        ),
+                        ConvBlock(
+                            filters=int(256 * width_multiplier),
+                            kernel_size=3,
+                            strides=1,
+                            activation=activation,
+                        ),
+                    ]
+                )
+            )
+
+            self.classification_preds.append(
+                keras.layers.Conv2D(
+                    filters=classes,
+                    kernel_size=1,
+                    strides=1,
+                    padding="same",
+                    bias_initializer=bias_initializer,
+                )
+            )
+            self.regression_preds.append(
+                keras.layers.Conv2D(
+                    filters=4,
+                    kernel_size=1,
+                    strides=1,
+                    padding="same",
+                    bias_initializer=bias_initializer,
+                )
+            )
+            self.objectness_preds.append(
+                keras.layers.Conv2D(
+                    filters=1,
+                    kernel_size=1,
+                    strides=1,
+                    padding="same",
+                )
+            )
+
+    def call(self, inputs, training=False):
+        outputs = []
+
+        for i, p_i in enumerate(inputs):
+            stem = self.stems[i](p_i)
+
+            classes = self.classification_convs[i](stem)
+            classes = self.classification_preds[i](classes)
+
+            boxes_feat = self.regression_convs[i](stem)
+            boxes = self.regression_preds[i](boxes_feat)
+            objectness = self.objectness_preds[i](boxes_feat)
+
+            output = tf.keras.layers.Concatenate(axis=-1)([boxes, objectness, classes])
```

Comment:
here's a question: should we actually concat these or should we just pass a dictionary around?